explorer/src/components/Staking/OperatorsList.tsx

```diff
@@ -84,7 +89,7 @@ export const OperatorsList: FC = () => {
     setAction({ type: OperatorActionType.None, operatorId: null, maxShares: null })
   }, [])
 
-  const { network, section } = useChains()
+  const { network } = useChains()
   const apolloClient = useApolloClient()
 
   const columns = useMemo(() => {
```

Comment:
There are too many columns, we need to figure out better visualization for the data. The table is close to using my entire screen. @DianaPertseva any ideas?
![image](https://github.com/user-attachments/assets/7f74ceff-16cd-4a54-bd37-4d2688f7ffb6)

Reply:
I'd remove the "deposits" and "withdrawals" columns or combine these two / show the up or down trend with a %. Similarly, for "My stake" -- I think it might be more suitable in the sidekick info about your stake. What do you think?

Reply:
These columns have been remove and a columns editor has been added to allow user to customize the table as they wish